Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -1753,6 +1755,7 @@ public void TestAsyncLoginTimeout()
         [Test, NonParallelizable]
         public void TestAsyncDefaultLoginTimeout()
         {
+            SnowflakeDbConnectionPool.SetPooling(false);
```

Comment:
here you don't clean up changes to pooling.
Maybe it would be good to add TearDown method to be executed after each of tests to set the default value of pooling.

Reply:
cause not all tests disable pooling, that's why it was so flaky